_community/utopianio.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Utopian.io 
+position: 6
+---
+
+Utopian Platform rewards everyone who contributes to open-source projects. The community includes free software enthusiast, moderators and supervisors who look after quality of contributions being posted by developers. 
+
+If you are interested in building STEEM projects on the top of Utopian Platform, feel free to use official API wrapper for nodeJS.
```

Comment:
Can it be made more clear that this is describing the "NPM module"? Either that or rename the "Official NPM module" link to be something that matches the description.